### ISSUE TITLE
Unable to create a "test" stream.

### ISSUE THREAD

**Issue by rahuld-innovalabs** [muxinc/mux-ruby#15]
### Environment

- ruby-2.6.3
- rails 6.0.0
- mux_ruby-1.4.0

### Current Behaviour

When working with non-production environments in our app like staging, testing etc., we want to create streams with test live streams so that we are not charged for testing purposes. According to mux docs here https://docs.mux.com/reference#create-a-live-stream we can do it by passing `test: true` in param while creating a live stream. Tried to do it in the following ways but none worked.

- `MuxRuby::CreateLiveStreamRequest.new(test: true)`
- `client.create_live_stream(create_live_stream_request, test: true)` 

### Expected Behavior

By any of the ways suggested above we should be able to create live streams with `test: true` param. These streams should have a mux logo while being watched and can not be broadcasted to, more than 5 minutes.

**Comment by GeneticGenesis** [comment]
Hey @rahuld-innovalabs, 

Thanks so much for pointing this out. I've updated our OpenAPI bindings to fix this. If you update your gemfile to depend on the following branch, you can pick up a fix for this:

```
gem 'mux_ruby', git: 'git://github.com/muxinc/mux-ruby.git', branch: 'test-mode'
```

A full fix will go out in 1.5.0 which I will release next week.

Thanks!

**Comment by rahuld-innovalabs** [comment]
Hey @GeneticGenesis,

Thank you so much for a quick fix until  the next version it out. :pray:

**Comment by GeneticGenesis** [comment]
Whoops! This was fixed in 1.5.0, released on May 7th!

https://rubygems.org/gems/mux_ruby/versions/1.5.0
